server/go.mod

```diff
@@ -3,10 +3,10 @@ module github.com/mattermost/mattermost/server/v8
 go 1.20
 
 require (
-	code.sajari.com/docconv/v2 v2.0.0-pre.3.0.20240118234718-6e3f8759fa34
+	code.sajari.com/docconv/v2 v2.0.0-pre.4
```

Comment:
Can you just confirm that pre.4 contains the fix in that commit? We need that for a vulnerability.

Reply:
It seems like v2.0.0-pre.4 is pointing exactly to that commit: https://github.com/sajari/docconv/releases/tag/v2.0.0-pre.4 @jupenur, I think it's pretty clear, but can you confirm?

Reply:
Thanks!